cfy_manager/components/base_component/base_component.py

```diff
@@ -51,6 +51,9 @@ def stop(self):
     def remove(self):
         pass
 
+    def upgrade(self):
```

Comment:
Great idea to put this function as part of the base component. It might serve us also in the future. 
Should we consider implementing it in the prometheus component too?

Reply:
Frankly, I'm not sure if this is such a great idea :)  I mean we'd be adding more complexity, which is not always desirable.  But currently I cannot think of a better solution.

As for Prometheus, I'll follow your suggestion in the next patch